### PR TITLE
Enhance search filter UI for JS disabled users

### PR DESCRIPTION
**Pull request by syeopite**
Allow JS disabled users to collapse the search filter list.

### CODE REVIEW

File: assets/css/default.css [hunk 1]
```diff
@@ -628,6 +628,31 @@ body.dark-theme {
...
+
+#filters > summary {
+  display: inline-block;
+  margin-bottom:15px;
+}
+
+#filters > summary {
+  display: inline-block;
+  margin-bottom:15px;
+}
```
Comment (saltycrys) [inline assets/css/default.css:648]: Please remove the duplicated styles and add a space.

File: src/invidious/views/search.ecr [hunk 2]
```diff
@@ -95,6 +87,7 @@
         <% end %>
     </div>
 </div>
+</details>
```
Comment (saltycrys) [inline src/invidious/views/search.ecr:90]: Please indent the `div`.

**Review by saltycrys — CHANGES_REQUESTED** [review]

**Review by saltycrys — CHANGES_REQUESTED** [review]
The indentation of the file is four spaces, so any changes should also be indented by four spaces.

**Comment by TheFrenchGhosty** [comment]
@saltycrys Can you please review this PR?

File: src/invidious/views/search.ecr [hunk 3]
```diff
@@ -141,4 +134,4 @@
         <% end %>
     </div>
 </div>
-<script src="/js/search.js?v=<%= ASSET_COMMIT %>"></script>
\ No newline at end of file
+<script src="/js/search.js?v=<%= ASSET_COMMIT %>"></script>
```
Comment (saltycrys) [inline src/invidious/views/search.ecr:137]: Please remove this.

**Review by saltycrys — CHANGES_REQUESTED** [review]

**Review by saltycrys — APPROVED** [review]

**Comment by TheFrenchGhosty** [comment]
Thank you! @syeopite 

Thanks for the review @saltycrys

**Pull request merged**